Makefile

```diff
@@ -2,7 +2,7 @@ VERSION = 1.4.0
 TAG = $(VERSION)
 PREFIX = nginx/nginx-prometheus-exporter
 # renovate: datasource=github-tags depName=golangci/golangci-lint
-GOLANGCI_LINT_VERSION = v1.64.8
+GOLANGCI_LINT_VERSION = v2.1.5
```

Comment:
https://github.com/nginx/nginx-prometheus-exporter/pull/1046/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L24 needs to be changed to `go run github.com/golangci/golangci-lint/v2/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) run --fix` (the v2 is required now after the version bump)

Reply:
Fixed in [abb5923](https://github.com/nginx/nginx-prometheus-exporter/pull/1046/commits/abb592397d85e9c735b862d434d030f91620a692)